README.md

```diff
@@ -65,48 +35,37 @@ In Firefox, it'll temporarily replace the existing Stylus extension. To restore
 
 </details>
 
-## Help
+## 🪄 Highlights
 
-* [Stylus help and FAQ in our Wiki](https://github.com/openstyles/stylus/wiki)
-* [Discussion section](https://add0n.com/stylus.html#reviews) of our representation on add0n.com
-* Discord: [![Discord][chat-image]][chat-link]
-
-[chat-image]: https://img.shields.io/discord/379521691774353408.svg
-[chat-link]: https://discordapp.com/widget?id=379521691774353408
+* Lightweight content script (10kB) runs in about a millisecond so it doesn't slow down web pages
+* In addition to the [userstyles.org](userstyles.org) site, styles with customizable parameters can also be installed from [.user.css](https://github.com/openstyles/stylus/wiki/Usercss) or .user.styl URLs.
+* Site styles can be discovered and previewed in the popup using inline search with screenshot thumbnails.
+* A backup feature which is compatible with other userstyles managers.
+* Configurable automatic update function for installed styles.
+* Customizable UI, optional layouts, and tweaks.
+* Two different optional code validators with user-configurable rules: CSSLint and Stylelint.
+  * Both validators use Web Worker API to run in a separate background thread inside the editor tab without blocking your interaction with the code.
+  * CSSLint is heavily modified compared to the effectively frozen original one and supports various CSS3 features as well as CSS4 Color and CSS Grid syntax.
 
-## Contributing
+## 💾 Install
 
-The source is hosted on [GitHub](https://github.com/openstyles/stylus) and pull requests are welcome.
+🦊 Firefox: [Firefox add-ons](https://addons.mozilla.org/firefox/addon/styl-us) • 🔵 Chrome: [Chrome Web Store](https://chrome.google.com/webstore/detail/stylus/clngdbkpkpeebahjckkjfobafhncgmne) • ⚙️ Beta: [Github Releases](https://github.com/openstyles/stylus/releases)
 
-You can help us translate the extension on [Transifex](https://explore.transifex.com/github-7/Stylus/).
+## ❤️ Contributing
+ 🛠 [Pull requests](https://github.com/openstyles/stylus/pulls) are welcome, 🌎 translation work can be done on [Transifex](https://explore.transifex.com/github-7/Stylus).
```

Comment:
The icon for PR makes sense, but the earth globe is too vague, so it's best to remove both.